kubernetes-examples/keycloak.yaml

```diff
@@ -31,7 +31,7 @@ spec:
     spec:
       containers:
       - name: keycloak
-        image: quay.io/keycloak/keycloak:18.0.0-SNAPSHOT
+        image: quay.io/keycloak/keycloak:999-SNAPSHOT
```

Comment:
```suggestion
        image: quay.io/keycloak/keycloak:latest
```

Reply:
Setting these to nightly in https://github.com/keycloak/keycloak-quickstarts/pull/310